gen/definitions/ext_community_set.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: Extcommunity Set
```

Comment:
Due to the YANG models having different lists for different types of extcommunity sets (opaque, cost, etc.) we would need to rename the resource to `Extcommunity Opaque Set`. Make sure you delete all the files created from the "old" name and then run `go generate` again.

Reply:
added the new yaml file for extcommunity_opaque_set